lib/recipeData.tsx

```diff
@@ -212,6 +213,26 @@ const callbackConfig = {
 
 const LoginWithOneTap = () => <StytchLogin config={sdkConfig} callbacks={callbackConfig} />;`,
   },
+  SMS_MFA: {
+    id: 'smsmfa',
+    title: 'Headless SDK + SMS MFA',
+    details:
+        'For developers that want full control over the user experience while minimizing backend code and session logic, you can interact directly with Stytch’s Headless SDK and API’s. ',
+    description: `In this example we use custom UI elements and backend API logic to implement a two factor authentication flow with Email magic links as the primary factor and SMS OTP as the secondary factor.`,
+    instructions: `To the right you'll see an email address entry form built within this example app, not using our pre-built UI. You'll start off the flow by using Email magic links as a primary factor, then you'll be prompted to register and authenticate via SMS OTP as a second factor in a multi factor authentication flow.`,
+    component: <LoginWithSMSMFA />,
+    products: [LoginProducts.EML, LoginProducts.SMS],
+    code: `// Send the Email magic link
+    await stytchClient.magicLinks.email.loginOrCreate({
```

Comment:
Why is there no example code for the sms mfa? Isn't that what we're trying to show?

Reply:
Trying to pattern match against the other recipes that are on the Stytch demo right now...all only show the code for the first auth factor of authentication (we thought it would make sense to keep it this way)